<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_changeKey</class>
 <widget class="QDialog" name="d_changeKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    background-color: white;
    border: 2px solid #868282;
    border-radius: 5px;
}

QLineEdit:hover {
    background-color: #F5EFEF;
    border: 2px solid #868282;
    border-radius: 5px;
}
#d_changeKey {
    border-image: url(:/bg/resource/bg_key.png);
}
QLabel {
    color: #535151;
    font-family: &quot;黑体&quot;;
}
#pushButton_changeKey_return{
    border-image: url(:/icons/resource/back.png);
}

#pushButton_changeKey_return:hover{
    border-image: url(:/icons/resource/back_h.png);
}
#pushButton_changeKey_ok{
    border-image: url(:/icons/resource/ok.png);
}

#pushButton_changeKey_ok:hover{
    border-image: url(:/icons/resource/ok_h.png);
}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_changeKey_oldKey">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_changeKey_key1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_changeKey_key2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changeKey_return">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changeKey_ok">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>640</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_changeKey_info">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>旧 密 码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新 密 码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
